training/distributed_training/pytorch/model_parallel/gpt2/smp-fine-tune-gpt-sharded-data-parallel.ipynb

```diff
@@ -0,0 +1,971 @@
+{
```

Comment:
<p>Set the model configuration. Specify one from <code>gpt2-30b</code>, <code>gpt2-xl</code> and <code>gpt2-small</code>, depending on the model you checkpointed and want to load for fine-tuning. This should match with the parameter specified to <code>model_name</code> in the <code>hyperparameters</code> dictionary in the previous code block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>